src/HyperFormula.ts

```diff
@@ -1820,6 +1835,8 @@ export class HyperFormula implements TypedEmitter {
    * @param {SimpleCellRange} source - range for a moved block
    * @param {SimpleCellAddress} destinationLeftCorner - upper left address of the target cell block
    *
+   * @throws [[NoSheetWithIdError]] when the given sheet ID does not exist
+   * @throws [[ExpectedValueOfTypeError]] if any of its basic type argument is of wrong type
    * @throws [[ExpectedValueOfTypeError]] if destinationLeftCorner or source are of wrong type
```

Comment:
This error type is listed twice? We should merge those

Reply:
@wojciechczerniak - done: 58d95fc